.gitignore

```diff
@@ -4,6 +4,8 @@
 *.png-
 *.txt
 
+*.bak
```

Comment:
Backup file extensions (`*.bak`, `*~`, ...) and also other patterns like `.DS_Store` can be added to your personal Git settings, so there is no need to add them to project settings like this one. Ideally `.gitignore` should only ignore patterns which are required by the project.